test/test-functions

```diff
@@ -1121,7 +1121,8 @@ install_iscsi() {
         image_install iscsi-iname iscsiadm iscsid iscsistart
         image_install -o "${ROOTLIBDIR:?}"/system/iscsi-{init,onboot,shutdown}.service
         image_install "${ROOTLIBDIR:?}"/system/iscsid.{service,socket}
-        image_install "${ROOTLIBDIR:?}"/system/iscsi.service
+        image_install -o "${ROOTLIBDIR:?}"/system/iscsi.service
```

Comment:
It seems open-iscsi.service and related scripts are necessary, no?

Reply:
It does not look like, not for this test - this is enough to make it run successfully:

```
[   33.534154] (ORAGE.sh)[398]: TEST-64-UDEV-STORAGE.service: Executing: /usr/lib/systemd/tests/testdata/units/TEST-64-UDEV-STORAGE.sh
[   33.674974] TEST-64-UDEV-STORAGE.sh[398]: + set -o pipefail
[   33.722938] TEST-64-UDEV-STORAGE.sh[398]: + udevadm settle
[   33.795175] TEST-64-UDEV-STORAGE.sh[398]: + udevadm control --log-level debug
[   33.862555] TEST-64-UDEV-STORAGE.sh[398]: + lsblk -a
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: fd0      2:0    1    4K  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop0    7:0    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop1    7:1    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop2    7:2    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop3    7:3    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop4    7:4    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop5    7:5    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop6    7:6    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: loop7    7:7    0    0B  0 loop
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: sda      8:0    0   64M  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: sdb      8:16   0   64M  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: sdc      8:32   0  150M  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: sdd      8:48   0   64M  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: sde      8:64   0  1.1G  0 disk
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: ├─sde1   8:65   0  128M  0 part
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: ├─sde2   8:66   0  768M  0 part /
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: ├─sde3   8:67   0  128M  0 part
[   33.992632] TEST-64-UDEV-STORAGE.sh[401]: └─sde4   8:68   0   98M  0 part
[   34.003317] TEST-64-UDEV-STORAGE.sh[398]: + echo 'Check if all symlinks under /dev/disk/ are valid (pre-test)'
[   34.003317] TEST-64-UDEV-STORAGE.sh[398]: Check if all symlinks under /dev/disk/ are valid (pre-test)
[   34.003317] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks
[   34.007214] TEST-64-UDEV-STORAGE.sh[402]: + set +x
[   42.737543] TEST-64-UDEV-STORAGE.sh[398]: + command -v testcase_iscsi_lvm
[   42.737543] TEST-64-UDEV-STORAGE.sh[398]: testcase_iscsi_lvm
[   42.737543] TEST-64-UDEV-STORAGE.sh[398]: + echo TEST_FUNCTION_NAME=testcase_iscsi_lvm
[   42.737543] TEST-64-UDEV-STORAGE.sh[398]: TEST_FUNCTION_NAME=testcase_iscsi_lvm
[   42.737543] TEST-64-UDEV-STORAGE.sh[398]: + testcase_iscsi_lvm
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local dev i label link lun_id mpoint target_name uuid
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local target_ip=127.0.0.1
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local target_port=3260
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local vgroup=iscsi_lvm9252
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks=()
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local expected_symlinks
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + devices=('/dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0' '/dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi1' '/dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi2' '/dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi3')
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + local devices
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + . /etc/os-release
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ PRETTY_NAME='Debian GNU/Linux trixie/sid'
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ NAME='Debian GNU/Linux'
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ VERSION_CODENAME=trixie
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ ID=debian
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ HOME_URL=https://www.debian.org/
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ SUPPORT_URL=https://www.debian.org/support
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: ++ BUG_REPORT_URL=https://bugs.debian.org/
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + [[ debian == \u\b\u\n\t\u ]]
[   42.792531] TEST-64-UDEV-STORAGE.sh[398]: + ls -l /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi1 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi2 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi3
[   42.852991] TEST-64-UDEV-STORAGE.sh[544]: lrwxrwxrwx 1 root root 9 May 27 10:12 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0 -> ../../sdc
[   42.852991] TEST-64-UDEV-STORAGE.sh[544]: lrwxrwxrwx 1 root root 9 May 27 10:12 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi1 -> ../../sdb
[   42.852991] TEST-64-UDEV-STORAGE.sh[544]: lrwxrwxrwx 1 root root 9 May 27 10:12 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi2 -> ../../sda
[   42.852991] TEST-64-UDEV-STORAGE.sh[544]: lrwxrwxrwx 1 root root 9 May 27 10:12 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi3 -> ../../sdd
[   42.859877] TEST-64-UDEV-STORAGE.sh[398]: + systemctl list-unit-files tgt.service
[   43.111558] TEST-64-UDEV-STORAGE.sh[545]: UNIT FILE   STATE    PRESET
[   43.111558] TEST-64-UDEV-STORAGE.sh[545]: tgt.service disabled enabled
[   43.117172] TEST-64-UDEV-STORAGE.sh[545]: 1 unit files listed.
[   43.121043] TEST-64-UDEV-STORAGE.sh[398]: + systemctl start tgt
[   45.645405] TEST-64-UDEV-STORAGE.sh[398]: + systemctl status tgt
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: ● tgt.service - (i)SCSI target daemon
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:      Loaded: loaded (/usr/lib/systemd/system/tgt.service; disabled; preset: enabled)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:      Active: active (running) since Mon 2024-05-27 10:12:42 UTC; 211ms ago
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:  Invocation: e4e151b437e74c3b9a35d2e54c5c8ce2
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:        Docs: man:tgtd(8)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:     Process: 557 ExecStartPost=/usr/sbin/tgtadm --op update --mode sys --name State -v offline (code=exited, status=0/SUCCESS)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:     Process: 559 ExecStartPost=/usr/sbin/tgt-admin -e -c /etc/tgt/targets.conf (code=exited, status=0/SUCCESS)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:     Process: 564 ExecStartPost=/usr/sbin/tgtadm --op update --mode sys --name State -v ready (code=exited, status=0/SUCCESS)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:    Main PID: 547 (tgtd)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:      Status: "Starting event loop..."
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:       Tasks: 1
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:      Memory: 4.9M (peak: 11.0M)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:         CPU: 1.673s
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:      CGroup: /system.slice/tgt.service
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]:              └─547 /usr/sbin/tgtd -f
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:41 H (tgtadm)[564]: Found cgroup2 on /sys/fs/cgroup/, full unified hierarchy
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:41 H (tgtadm)[564]: Found cgroup2 on /sys/fs/cgroup/, full unified hierarchy
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:41 H (tgtadm)[564]: tgt.service: Executing: /usr/sbin/tgtadm --op update --mode sys --name State -v ready
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Child 564 belongs to tgt.service.
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Control process exited, code=exited, status=0/SUCCESS (success)
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Got final SIGCHLD for state start-post.
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Changed start-post -> running
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Job 312 tgt.service/start finished, result=done
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: Started tgt.service.
[   45.888899] TEST-64-UDEV-STORAGE.sh[566]: May 27 10:12:42 H systemd[1]: tgt.service: Failed to send unit change signal for tgt.service: Connection reset by peer
[   45.904271] TEST-64-UDEV-STORAGE.sh[398]: + echo 'iSCSI LUNs backed by devices'
[   45.904271] TEST-64-UDEV-STORAGE.sh[398]: iSCSI LUNs backed by devices
[   45.904271] TEST-64-UDEV-STORAGE.sh[398]: + target_name=iqn.2021-09.com.example:iscsi.test
[   45.904271] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode target --tid=1 --targetname iqn.2021-09.com.example:iscsi.test
[   45.933793] TEST-64-UDEV-STORAGE.sh[398]: + (( i = 0 ))
[   45.935899] TEST-64-UDEV-STORAGE.sh[398]: + (( i < 4 ))
[   45.937614] TEST-64-UDEV-STORAGE.sh[398]: + lun_id=1
[   45.938453] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 1 --lun 1 -b /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0
[   45.989381] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 1 --lun 1
[   46.026723] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$lun_id")
[   46.027661] TEST-64-UDEV-STORAGE.sh[398]: + (( i++ ))
[   46.027661] TEST-64-UDEV-STORAGE.sh[398]: + (( i < 4 ))
[   46.027661] TEST-64-UDEV-STORAGE.sh[398]: + lun_id=2
[   46.029398] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 1 --lun 2 -b /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi1
[   46.075961] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 1 --lun 2
[   46.112382] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$lun_id")
[   46.112382] TEST-64-UDEV-STORAGE.sh[398]: + (( i++ ))
[   46.113754] TEST-64-UDEV-STORAGE.sh[398]: + (( i < 4 ))
[   46.113754] TEST-64-UDEV-STORAGE.sh[398]: + lun_id=3
[   46.113754] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 1 --lun 3 -b /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi2
[   46.158569] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 1 --lun 3
[   46.194290] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$lun_id")
[   46.195201] TEST-64-UDEV-STORAGE.sh[398]: + (( i++ ))
[   46.195201] TEST-64-UDEV-STORAGE.sh[398]: + (( i < 4 ))
[   46.195201] TEST-64-UDEV-STORAGE.sh[398]: + lun_id=4
[   46.195201] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 1 --lun 4 -b /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi3
[   46.239182] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 1 --lun 4
[   46.275170] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$lun_id")
[   46.276380] TEST-64-UDEV-STORAGE.sh[398]: + (( i++ ))
[   46.276380] TEST-64-UDEV-STORAGE.sh[398]: + (( i < 4 ))
[   46.276380] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op bind --mode target --tid 1 -I ALL
[   46.313462] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode discoverydb --type sendtargets --portal 127.0.0.1 --discover
[   48.604649] TEST-64-UDEV-STORAGE.sh[642]: 127.0.0.1:3260,1 iqn.2021-09.com.example:iscsi.test
[   48.608919] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.test --portal 127.0.0.1:3260 --login
[   48.856135] kernel: sd 3:0:0:1: Power-on or device reset occurred
[   48.901958] kernel: sd 3:0:0:2: Power-on or device reset occurred
[   48.953303] kernel: sd 3:0:0:3: Power-on or device reset occurred
[   49.012329] TEST-64-UDEV-STORAGE.sh[659]: Logging in to [iface: default, target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260]
[   49.012329] TEST-64-UDEV-STORAGE.sh[659]: Login to [iface: default, target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260] successful.
[   49.023838] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.test-lun-4
[   49.039619] kernel: sd 3:0:0:4: Power-on or device reset occurred
[   49.215212] iscsid[650]: Could not set session1 priority. READ/WRITE throughout and latency could be affected.
[   49.215299] iscsid[650]: Connection1:0 to [target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260] through [iface: default] is operational now
[   49.750655] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks
[   49.761005] TEST-64-UDEV-STORAGE.sh[677]: + set +x
[   60.664408] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_units
[   60.676607] TEST-64-UDEV-STORAGE.sh[851]: + set +x
[   95.577187] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.test --portal 127.0.0.1:3260 --logout
[   96.047268] TEST-64-UDEV-STORAGE.sh[1395]: Logging out of session [sid: 1, target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260]
[   96.047268] TEST-64-UDEV-STORAGE.sh[1395]: Logout of [sid: 1, target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260] successful.
[   96.053590] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op delete --mode target --tid=1
[   96.394358] iscsid[650]: Connection1:0 to [target: iqn.2021-09.com.example:iscsi.test, portal: 127.0.0.1,3260] through [iface: default] is shutdown.
[   96.549773] TEST-64-UDEV-STORAGE.sh[398]: + echo 'iSCSI LUNs backed by files + LVM'
[   96.549773] TEST-64-UDEV-STORAGE.sh[398]: iSCSI LUNs backed by files + LVM
[   96.549773] TEST-64-UDEV-STORAGE.sh[398]: + target_name=iqn.2021-09.com.example:iscsi.lvm.test
[   96.564926] TEST-64-UDEV-STORAGE.sh[1405]: ++ mktemp -d /iscsi_storeXXX
[   96.572819] TEST-64-UDEV-STORAGE.sh[398]: + mpoint=/iscsi_storeOe4
[   96.572819] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks=()
[   96.572819] TEST-64-UDEV-STORAGE.sh[398]: + mkfs.ext4 -L iscsi_store /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0
[   96.580613] TEST-64-UDEV-STORAGE.sh[1410]: mke2fs 1.47.1 (20-May-2024)
[   96.635086] TEST-64-UDEV-STORAGE.sh[1410]: Discarding device blocks: done         
[   96.646663] TEST-64-UDEV-STORAGE.sh[1410]: Creating filesystem with 153600 1k blocks and 38456 inodes
[   96.646663] TEST-64-UDEV-STORAGE.sh[1410]: Filesystem UUID: 5aec5ece-f75c-420b-9c99-c2c1aae9f39d
[   96.646663] TEST-64-UDEV-STORAGE.sh[1410]: Superblock backups stored on blocks:
[   96.646663] TEST-64-UDEV-STORAGE.sh[1410]: 	8193, 24577, 40961, 57345, 73729
[   96.654425] TEST-64-UDEV-STORAGE.sh[1410]: Allocating group tables: done 
[   96.654425] TEST-64-UDEV-STORAGE.sh[1410]: Writing inode tables: done 
[   96.737968] TEST-64-UDEV-STORAGE.sh[1410]: Creating journal (4096 blocks): done
[   96.830032] TEST-64-UDEV-STORAGE.sh[1410]: Writing superblocks and filesystem accounting information: done 
[   96.871863] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0
[   97.006261] TEST-64-UDEV-STORAGE.sh[398]: + mount /dev/disk/by-id/scsi-0systemd_foobar_deadbeefiscsi0 /iscsi_storeOe4
[   97.255110] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   97.255110] TEST-64-UDEV-STORAGE.sh[398]: + dd if=/dev/zero of=/iscsi_storeOe4/lun1.img bs=1M count=32
[   97.679005] TEST-64-UDEV-STORAGE.sh[1418]: 32+0 records in
[   97.679005] TEST-64-UDEV-STORAGE.sh[1418]: 32+0 records out
[   97.686128] TEST-64-UDEV-STORAGE.sh[1418]: 33554432 bytes (34 MB, 32 MiB) copied, 0.340234 s, 98.6 MB/s
[   97.691070] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   97.691767] TEST-64-UDEV-STORAGE.sh[398]: + dd if=/dev/zero of=/iscsi_storeOe4/lun2.img bs=1M count=32
[   98.095616] TEST-64-UDEV-STORAGE.sh[1419]: 32+0 records in
[   98.095616] TEST-64-UDEV-STORAGE.sh[1419]: 32+0 records out
[   98.103487] TEST-64-UDEV-STORAGE.sh[1419]: 33554432 bytes (34 MB, 32 MiB) copied, 0.352875 s, 95.1 MB/s
[   98.107093] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   98.111707] TEST-64-UDEV-STORAGE.sh[398]: + dd if=/dev/zero of=/iscsi_storeOe4/lun3.img bs=1M count=32
[   98.619973] TEST-64-UDEV-STORAGE.sh[1420]: 32+0 records in
[   98.619973] TEST-64-UDEV-STORAGE.sh[1420]: 32+0 records out
[   98.625420] TEST-64-UDEV-STORAGE.sh[1420]: 33554432 bytes (34 MB, 32 MiB) copied, 0.461442 s, 72.7 MB/s
[   98.630009] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   98.630009] TEST-64-UDEV-STORAGE.sh[398]: + dd if=/dev/zero of=/iscsi_storeOe4/lun4.img bs=1M count=32
[   99.081079] TEST-64-UDEV-STORAGE.sh[1421]: 32+0 records in
[   99.081079] TEST-64-UDEV-STORAGE.sh[1421]: 32+0 records out
[   99.085731] TEST-64-UDEV-STORAGE.sh[1421]: 33554432 bytes (34 MB, 32 MiB) copied, 0.402335 s, 83.4 MB/s
[   99.089358] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode target --tid=2 --targetname iqn.2021-09.com.example:iscsi.lvm.test
[   99.129944] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   99.134506] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 2 --lun 1 -b /iscsi_storeOe4/lun1.img
[   99.182522] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 2 --lun 1
[   99.219134] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$i")
[   99.219969] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   99.219969] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 2 --lun 2 -b /iscsi_storeOe4/lun2.img
[   99.267097] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 2 --lun 2
[   99.304418] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$i")
[   99.305959] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   99.305959] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 2 --lun 3 -b /iscsi_storeOe4/lun3.img
[   99.351283] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 2 --lun 3
[   99.388008] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$i")
[   99.389335] TEST-64-UDEV-STORAGE.sh[398]: + for i in {1..4}
[   99.389335] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op new --mode logicalunit --tid 2 --lun 4 -b /iscsi_storeOe4/lun4.img
[   99.433092] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op update --mode logicalunit --tid 2 --lun 4
[   99.469799] TEST-64-UDEV-STORAGE.sh[398]: + expected_symlinks+=("/dev/disk/by-path/ip-$target_ip:$target_port-iscsi-$target_name-lun-$i")
[   99.470677] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op bind --mode target --tid 2 -I ALL
[   99.507886] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode discoverydb --type sendtargets --portal 127.0.0.1 --discover
[   99.637340] TEST-64-UDEV-STORAGE.sh[1496]: 127.0.0.1:3260,1 iqn.2021-09.com.example:iscsi.lvm.test
[   99.641830] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.lvm.test --portal 127.0.0.1:3260 --login
[   99.883065] kernel: sd 3:0:0:1: Power-on or device reset occurred
[   99.930574] kernel: sd 3:0:0:2: Power-on or device reset occurred
[  100.014933] kernel: sd 3:0:0:3: Power-on or device reset occurred
[  100.074961] TEST-64-UDEV-STORAGE.sh[1497]: Logging in to [iface: default, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260]
[  100.074961] TEST-64-UDEV-STORAGE.sh[1497]: Login to [iface: default, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] successful.
[  100.102072] kernel: sd 3:0:0:4: Power-on or device reset occurred
[  100.188800] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4
[  100.398751] iscsid[650]: Could not set session2 priority. READ/WRITE throughout and latency could be affected.
[  100.398829] iscsid[650]: Connection2:0 to [target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] through [iface: default] is operational now
[  101.288718] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks
[  101.294612] TEST-64-UDEV-STORAGE.sh[1515]: + set +x
[  112.203309] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_units
[  112.215999] TEST-64-UDEV-STORAGE.sh[1689]: + set +x
[  136.479883] kernel: hrtimer: interrupt took 12405680 ns
[  146.788055] TEST-64-UDEV-STORAGE.sh[398]: + lvm pvcreate -y /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4
[  147.433941] TEST-64-UDEV-STORAGE.sh[2233]:   Physical volume "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1" successfully created.
[  147.446738] TEST-64-UDEV-STORAGE.sh[2233]:   Physical volume "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2" successfully created.
[  147.456637] TEST-64-UDEV-STORAGE.sh[2233]:   Physical volume "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3" successfully created.
[  147.467539] TEST-64-UDEV-STORAGE.sh[2233]:   Physical volume "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4" successfully created.
[  147.558800] TEST-64-UDEV-STORAGE.sh[398]: + lvm pvs
[  147.940902] TEST-64-UDEV-STORAGE.sh[2234]:   PV         VG Fmt  Attr PSize  PFree
[  147.940902] TEST-64-UDEV-STORAGE.sh[2234]:   /dev/sdf      lvm2 ---  32.00m 32.00m
[  147.940902] TEST-64-UDEV-STORAGE.sh[2234]:   /dev/sdg      lvm2 ---  32.00m 32.00m
[  147.940902] TEST-64-UDEV-STORAGE.sh[2234]:   /dev/sdh      lvm2 ---  32.00m 32.00m
[  147.940902] TEST-64-UDEV-STORAGE.sh[2234]:   /dev/sdi      lvm2 ---  32.00m 32.00m
[  148.043242] TEST-64-UDEV-STORAGE.sh[398]: + lvm vgcreate iscsi_lvm9252 -y /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4
[  149.349960] TEST-64-UDEV-STORAGE.sh[2247]:   Volume group "iscsi_lvm9252" successfully created
[  149.431404] TEST-64-UDEV-STORAGE.sh[398]: + lvm vgs
[  149.920951] TEST-64-UDEV-STORAGE.sh[2256]:   VG            #PV #LV #SN Attr   VSize   VFree
[  149.920951] TEST-64-UDEV-STORAGE.sh[2256]:   iscsi_lvm9252   4   0   0 wz--n- 112.00m 112.00m
[  150.003060] TEST-64-UDEV-STORAGE.sh[398]: + lvm vgchange -ay iscsi_lvm9252
[  150.262955] TEST-64-UDEV-STORAGE.sh[2257]:   0 logical volume(s) in volume group "iscsi_lvm9252" now active
[  150.357144] TEST-64-UDEV-STORAGE.sh[398]: + lvm lvcreate -y -L 4M iscsi_lvm9252 -n mypart1
[  151.029435] TEST-64-UDEV-STORAGE.sh[2258]:   Logical volume "mypart1" created.
[  151.122502] TEST-64-UDEV-STORAGE.sh[398]: + lvm lvcreate -y -L 8M iscsi_lvm9252 -n mypart2
[  152.787721] TEST-64-UDEV-STORAGE.sh[2264]:   Logical volume "mypart2" created.
[  152.931132] TEST-64-UDEV-STORAGE.sh[398]: + lvm lvs
[  153.739751] TEST-64-UDEV-STORAGE.sh[2279]:   LV      VG            Attr       LSize Pool Origin Data%  Meta%  Move Log Cpy%Sync Convert
[  153.752894] TEST-64-UDEV-STORAGE.sh[2279]:   mypart1 iscsi_lvm9252 -wi-a----- 4.00m
[  153.752894] TEST-64-UDEV-STORAGE.sh[2279]:   mypart2 iscsi_lvm9252 -wi-a----- 8.00m
[  153.844497] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/iscsi_lvm9252/mypart1 /dev/iscsi_lvm9252/mypart2
[  154.092681] TEST-64-UDEV-STORAGE.sh[398]: + mkfs.ext4 -L mylvpart1 /dev/iscsi_lvm9252/mypart1
[  154.175016] TEST-64-UDEV-STORAGE.sh[2289]: mke2fs 1.47.1 (20-May-2024)
[  154.233245] TEST-64-UDEV-STORAGE.sh[2289]: Creating filesystem with 4096 1k blocks and 1024 inodes
[  154.236755] TEST-64-UDEV-STORAGE.sh[2289]: Allocating group tables: done
[  154.248594] TEST-64-UDEV-STORAGE.sh[2289]: Writing inode tables: done
[  154.334582] TEST-64-UDEV-STORAGE.sh[2289]: Creating journal (1024 blocks): done
[  154.390491] TEST-64-UDEV-STORAGE.sh[2289]: Writing superblocks and filesystem accounting information: done
[  154.401155] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/disk/by-label/mylvpart1
[  154.614956] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks /dev/disk /dev/iscsi_lvm9252
[  154.623079] TEST-64-UDEV-STORAGE.sh[2292]: + set +x
[  166.855044] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_units
[  166.924436] TEST-64-UDEV-STORAGE.sh[2490]: + set +x
[  207.821530] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.lvm.test --portal 127.0.0.1:3260 --logout
[  208.260399] TEST-64-UDEV-STORAGE.sh[3128]: Logging out of session [sid: 2, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260]
[  208.260399] TEST-64-UDEV-STORAGE.sh[3128]: Logout of [sid: 2, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] successful.
[  208.272903] TEST-64-UDEV-STORAGE.sh[398]: + dmsetup remove_all --deferred
[  208.478424] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 --removed /dev/iscsi_lvm9252 /dev/disk/by-label/mylvpart1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4
[  208.503904] iscsid[650]: Connection2:0 to [target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] through [iface: default] is shutdown.
[  208.600156] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks /dev/disk
[  208.608808] TEST-64-UDEV-STORAGE.sh[3139]: + set +x
[  217.182119] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_units
[  217.186106] TEST-64-UDEV-STORAGE.sh[3281]: + set +x
[  244.337675] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode discoverydb --type sendtargets --portal 127.0.0.1 --discover
[  244.511968] TEST-64-UDEV-STORAGE.sh[3709]: 127.0.0.1:3260,1 iqn.2021-09.com.example:iscsi.lvm.test
[  244.519087] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.lvm.test --portal 127.0.0.1:3260 --login
[  244.839352] kernel: sd 3:0:0:1: Power-on or device reset occurred
[  244.879003] kernel: sd 3:0:0:2: Power-on or device reset occurred
[  244.950244] kernel: sd 3:0:0:3: Power-on or device reset occurred
[  245.002725] TEST-64-UDEV-STORAGE.sh[3710]: Logging in to [iface: default, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260]
[  245.002725] TEST-64-UDEV-STORAGE.sh[3710]: Login to [iface: default, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] successful.
[  245.014329] TEST-64-UDEV-STORAGE.sh[398]: + udevadm wait --settle --timeout=30 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-1 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-2 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-3 /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2021-09.com.example:iscsi.lvm.test-lun-4 /dev/iscsi_lvm9252/mypart1 /dev/iscsi_lvm9252/mypart2 /dev/disk/by-label/mylvpart1
[  245.034303] kernel: sd 3:0:0:4: Power-on or device reset occurred
[  245.527320] iscsid[650]: Could not set session3 priority. READ/WRITE throughout and latency could be affected.
[  245.527636] iscsid[650]: Connection3:0 to [target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] through [iface: default] is operational now
[  245.820710] lvm[3734]: PV /dev/sdh online, VG iscsi_lvm9252 incomplete (need 3).
[  245.827124] lvm[3733]: PV /dev/sdg online, VG iscsi_lvm9252 incomplete (need 2).
[  245.829885] lvm[3735]: PV /dev/sdf online, VG iscsi_lvm9252 incomplete (need 1).
[  245.869831] lvm[3736]: PV /dev/sdi online, VG iscsi_lvm9252 is complete.
[  248.225009] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks /dev/disk /dev/iscsi_lvm9252
[  248.240968] TEST-64-UDEV-STORAGE.sh[3752]: + set +x
[  260.564694] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_units
[  260.610429] TEST-64-UDEV-STORAGE.sh[3950]: + set +x
[  300.603382] TEST-64-UDEV-STORAGE.sh[398]: + iscsiadm --mode node --targetname iqn.2021-09.com.example:iscsi.lvm.test --portal 127.0.0.1:3260 --logout
[  300.998471] TEST-64-UDEV-STORAGE.sh[4588]: Logging out of session [sid: 3, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260]
[  300.998471] TEST-64-UDEV-STORAGE.sh[4588]: Logout of [sid: 3, target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] successful.
[  301.004224] TEST-64-UDEV-STORAGE.sh[398]: + tgtadm --lld iscsi --op delete --mode target --tid=2
[  301.146437] TEST-64-UDEV-STORAGE.sh[398]: + umount /iscsi_storeOe4
[  301.550311] TEST-64-UDEV-STORAGE.sh[398]: + rm -rf /iscsi_storeOe4
[  301.614117] TEST-64-UDEV-STORAGE.sh[398]: + udevadm settle
[  301.626889] iscsid[650]: Connection3:0 to [target: iqn.2021-09.com.example:iscsi.lvm.test, portal: 127.0.0.1,3260] through [iface: default] is shutdown.
[  301.715804] TEST-64-UDEV-STORAGE.sh[398]: + echo 'Check if all symlinks under /dev/disk/ are valid (post-test)'
[  301.719212] TEST-64-UDEV-STORAGE.sh[398]: Check if all symlinks under /dev/disk/ are valid (post-test)
[  301.719212] TEST-64-UDEV-STORAGE.sh[398]: + helper_check_device_symlinks
[  301.734671] TEST-64-UDEV-STORAGE.sh[4598]: + set +x
[  311.678374] TEST-64-UDEV-STORAGE.sh[398]: + udevadm control --log-level info
[  311.746808] TEST-64-UDEV-STORAGE.sh[398]: + systemctl status systemd-udevd
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: ● systemd-udevd.service - Rule-based Manager for Device Events and Files
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:      Loaded: loaded (/usr/lib/systemd/system/systemd-udevd.service; static)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:     Drop-In: /usr/lib/systemd/system/systemd-udevd.service.d
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:              └─syscall-architecture.conf
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:      Active: active (running) since Mon 2024-05-27 10:12:20 UTC; 4min 47s ago
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:  Invocation: 1af1224efa3e40debb03a131fd394ca7
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: TriggeredBy: ● systemd-udevd-control.socket
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:              ● systemd-udevd-kernel.socket
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:        Docs: man:systemd-udevd.service(8)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:              man:udev(7)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:    Main PID: 344 (systemd-udevd)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:      Status: "Processing with 10 children at max"
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:       Tasks: 1
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:      Memory: 6.5M (peak: 25.9M)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:         CPU: 27.748s
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:      CGroup: /system.slice/systemd-udevd.service
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:              └─udev
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]:                └─344 /usr/lib/systemd/systemd-udevd
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:16:57 H systemd-udevd[344]: No events are queued, removing /run/udev/queue.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:16:58 H systemd-udevd[344]: Received udev control message (PING)
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Cleanup idle workers
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Worker [4593] exited.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Worker [4589] exited.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Worker [4592] exited.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Worker [4590] exited.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:00 H systemd-udevd[344]: Worker [4591] exited.
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:04 H systemd-udevd[344]: Cleanup idle workers
[  312.061412] TEST-64-UDEV-STORAGE.sh[4757]: May 27 10:17:08 H systemd-udevd[344]: Received udev control message (SET_LOG_LEVEL), setting log_level=6
[  312.079435] TEST-64-UDEV-STORAGE.sh[398]: + touch /testok
[  317.990164] iscsid[651]: semop down failed 22
No journals were saved
------ testcase_iscsi_lvm: END (PASS) ------
Passed tests: 1
    * testcase_iscsi_lvm
Skipped tests: 0
    * 
Failed tests: 0
    * 
TEST-64-UDEV-STORAGE RUN: systemd-udev storage tests [OK]
```